OJP/OJP_Trips.xsd

```diff
@@ -129,9 +134,9 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:group ref="BaseTripMobilityFilterGroup"/>
-			<xs:element name="LevelEntrance" type="xs:boolean" default="false" minOccurs="0">
+			<xs:element name="LevelEntranceOrBoardingAid" type="xs:boolean" default="false" minOccurs="0">
```

Comment:
In my experience it makes a difference if there is a LevelEntrance or a BoardingAid. A LevelEntrance allows a self-reliant usage of the vehicle while a BoardingAid may involve other people (personnel) to arrange the boarding. Such an assisted boarding is not regarded as a a self-reliant usage of the vehicle.

Reply:
@JonaVBB This is a very pertinent remark. It means that LevelEntrance is generally preferred over LevelEntranceOrBoardingAid, and we should be able to take this into account. 

I'd propose to have both LevelEntrance and LevelEntranceOrBoardingAid as strict filter options. (LevelEntranceOrBoardingAid being more important since it allows for finding the routes feasible for an unassisted wheelchair)

For the AccessibilityPreferences, we need to add levelEntrance. (levelEntronceOrBoardingAid may still be useful, but less important)

If you agree, I'll make these changes.

Reply:
Totally agree.